guidelines/sc/21/text-spacing.html

```diff
@@ -14,5 +14,8 @@ <h4>Text Spacing</h4>
    </ul>
  
  <p>Exception: Human languages and scripts that do not make use of one or more of these text style properties in written text can conform using only the properties that exist for that combination of language and script.</p>
+
+ <p class="note">It is not required that content use the values specified. The requirement is to ensure no loss of content or functionality when a user overrides the authored styles with the values.</p>
+ <p class="note">Some human languages or writing systems have different needs and may adjust other values (such as paragraph start indent) in order to improve legibility. This success criterion might not be sufficient for those needs.</p>
```

Comment:
In our TPAC meeting, I18N suggests the following text to replace this paragraph, which incorporates some text from the understanding document (slightly modified)

```suggestion
<aside class="note">
<p>Some human languages or writing systems have different needs and may adjust other values (such as paragraph start indent) in order to improve legibility. This success criterion might not be sufficient for those needs. The exception in this SC allows authors to ignore text style properties which are inapplicable to the combination of language and script being used.</p>

<p>It is beneficial for users if authors use any locally available guidance for improving readability in the local language or writing system. If the user chooses to go beyond the metrics specified, any resulting loss of content or functionality is the user's responsibility.</p>
</aside>
```

Reply:
Hi @aphillips, it's getting pretty long now, it's about 25% more words than the SC itself. 
The notes should only be to clarify the normative requirement.

Can we remove the sentence starting "The exception" as it replicates the SC text?

Also, the last paragraph is adding a beneficial other thing for authors to do. The place for that is the understanding doc, can we remove that paragraph from the notes?

Reply:
I18N discussed in our WG call today (2023-09-21). We think it is okay to remove the "The exception..." sentence but we think that our second paragraph is important.